src/api.rs

```diff
@@ -73,12 +73,22 @@ pub struct Tx {
     pub fee: u64,
 }
 
-#[derive(Deserialize, Clone, Debug)]
+#[derive(Deserialize, Clone, Debug, PartialEq)]
 pub struct BlockTime {
     pub timestamp: u64,
     pub height: u32,
 }
 
+#[derive(Debug, Clone, Deserialize, PartialEq)]
+pub struct BlockSummary {
+    pub id: BlockHash,
+    #[serde(flatten)]
+    pub time: BlockTime,
+    /// Hash of the previous block, will be `None` for the genesis block.
+    pub previousblockhash: Option<bitcoin::BlockHash>,
+    pub merkle_root: bitcoin::TxMerkleNode,
+}
```

Comment:
To recreate `bdk_core::BlockId` from this we also need the  block height in the summary..

Reply:
It's in the `time` field.